bootstrap_cfn/fab_tasks.py

```diff
@@ -89,6 +89,8 @@ def _validate_fabric_env():
     if env.config is None:
         print "\n[ERROR] Please specify a config file, e.g 'config:/tmp/sample-application.yaml'"
         sys.exit(1)
+    elif not os.path.isfile(env.config):
+        sys.exit("\n[ERROR] Config file %s does not exist" % str(env.config))
```

Comment:
This doesn't follow the same pattern as the others - if you think this pattern is better (and it looks like it) then can you introduce another commit to change all the other ones in this function to follow the same pattern.